packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -707,15 +707,9 @@
 								{param ref: 'sortingButton' /}
 								{param spritemap: $spritemap /}
 								{param title kind="text"}
-									{if $sortingOrder == 'desc'}
-										{msg desc=""}
-											descending
-										{/msg}
-									{else}
-										{msg desc=""}
-											ascending
-										{/msg}
-									{/if}
+									{msg desc=""}
+										reverse-sort-direction
```

Comment:
hey @carloslancha I'm not sure if we should also apply this same text to when the button is disabled, it's currently `order` (https://github.com/liferay/clay/pull/1698/files#diff-54445d58e09b53d03b6d0558828824f6R726).

hey @victorvalle you can check this out.